kde-apps/knights/knights-9999.ebuild

```diff
@@ -19,21 +19,20 @@ KEYWORDS=""
 IUSE="speech"
 
 DEPEND="
-	>=dev-qt/qtbase-${QTMIN}:6[concurrent,dbus,gui,network,widgets]
+	>=dev-qt/qt5compat-${QTMIN}:6
+	>=dev-qt/qtbase-${QTMIN}:6[concurrent,gui,widgets]
```

Comment:
Still linking against Qt6DBus:
`    libQt6DBus.so.6 => /usr/lib64/libQt6DBus.so.6`

Still linking against Qt6Widgets:

`    libQt6Widgets.so.6 => /usr/lib64/libQt6Widgets.so.6`

Reply:
Keeping with the same rationale as with https://github.com/gentoo/kde/pull/972#discussion_r1505394573

Will remove the change.